package.json

```diff
@@ -19,11 +19,7 @@
     "lint": "pnpm -r --parallel run lint",
     "format": "pnpm -r --parallel run format"
   },
-  "packageManager": "pnpm@9.1.4",
-  "workspaces": [
```

Comment:
I think using `workspaces` could be beneficial for this monorepo proj, what do you think?

Reply:
I removed it because we already have a `pnpm-workspace` file so I think we dont need this in package.json